### ISSUE TITLE
Unable to have nodes with >25 edges?

### ISSUE THREAD

**Issue by elishillman** [imohitmayank/jaal#58]
I am attempting to make a large network with nodes that extend to many others. I have found that when a node has over 25 edges, the entire network disappears when I attempt to run Jaal. I am wondering if this is a known issue?

**Comment by imohitmayank** [comment]
Hey @elishillman I have not observed such issues. Can you share some more details about the data size (#nodes, #edges, etc).

**Comment by elishillman** [comment]
Yes. I am currently attempting to connect TV shows and cast and crew that worked on the show. I have collected ~60 cast/crew members per show and thus have ~60 edges extending from each show node. However, I have found that once I surpass 25 edges extending from a single node, the entirety of the graph disappears once I run the program. I have tested and I am able to view the graph when I reduce my data size to 25 cast/crew members per show, however I would of course like to include the entire data set.

**Comment by elishillman** [comment]
Please let me know if you want more direct info... screenshots, I send you the edge df, etc. I have continued testing and cannot figure out a solution. It seems to be capped at 25.

**Comment by varyemezzz** [comment]
Is there an update regarding this issue?
